packages/vue-components/src/PageNavButton.vue

```diff
@@ -19,30 +16,16 @@ import $ from './utils/NodeList';
 export default {
   data() {
     return {
-      hasPageNav: false,
-      hasIdentifier: false,
+      portalName: undefined,
       show: false,
-      src: '',
     };
   },
   computed: {
     showPageNav() {
-      return this.show && this.hasPageNav;
+      return this.show && this.portalName;
```

Comment:
I believe we need a separate variable to check for page nav as the identifier `page-nav` can exist without a page nav. 
For exmaple,
```
  <nav id="page-nav" class="fixed-header-padding">
    <div class="nav-component slim-scroll">
      {{ pageNav }}
    </div>
  </nav>
```
The `nav` element will always be present even if `{{pageNav}}` is empty.

This is causing the `page-nav-button` to show up on [pages](https://deploy-preview-1556--markbind-master.netlify.app/userguide/gettingstarted) without page nav.

![image](https://user-images.githubusercontent.com/47470981/116655581-ec1a6d00-a9bd-11eb-9d90-5c86d033ae59.png)

Reply:
reverted temporarily